genon/preprocessor/facade/intelligent_processor.py

```
@@ -1824,10 +1832,25 @@ async def __call__(self, request: Request, file_path: str, **kwargs: dict):
 
         vectors = []
         if len(chunks) >= 1:
-            vectors: list[dict] = await self.compose_vectors(document, chunks, file_path, request, **kwargs)
+            vectors: list[dict] = await self.compose_vectors(
+                document, chunks, file_path, request,
+                converted_pdf_path=converted_pdf_path,
+                **kwargs,
+            )
         else:
             raise GenosServiceException(1, f"chunk length is 0")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)